interfaces/builtin/rawusb.go

```diff
@@ -0,0 +1,43 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/snapcore/snapd/interfaces"
+)
+
+const rawusbConnectedPlugAppArmor = `
+# Until we have proper device assignment, allow access to all usb devices
+/dev/bus/usb/[0-9]{3}/[0-9]{3} rw,
```

Comment:
This rule does not compile. Please adjust. I suspect you are trying to use a regex here but apparmor syntax is a bit different. I suspect this will do what you want:
```
/dev/bus/usb/[0-9][0-9][0-9]/[0-9][0-9][0-9] rw,
```